### PR TITLE
Add instructions to enable WSL 2

### PR DESCRIPTION
**Pull request by andoriyu**
I've recently tried to install NixOS on a fresh installation of Windows 11. I couldn't import tarball until I install WSL either with another distro or no distro. `wsl.exe` doesn't even print `--import` as an option until it's done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,15 @@ Modules for running NixOS on the Windows Subsystem for Linux
 
 ## Quick Start
 
-1. Download `nixos-wsl.tar.gz` from [the latest release](https://github.com/nix-community/NixOS-WSL/releases/latest).
+0. Enable WSL if you haven’t done so:
```
Comment (SuperSandro2000) [inline README.md:14]:
```suggestion
1. Enable WSL if you haven’t done so:
```

File: README.md [hunk 2]
```diff
@@ -11,9 +11,15 @@ Modules for running NixOS on the Windows Subsystem for Linux
...
+  - ```powershell
+    wsl --install --no-distribution
+    ```
+
+2. Download `nixos-wsl.tar.gz` from [the latest release](https://github.com/nix-community/NixOS-WSL/releases/latest).
+
+3. Import the tarball into WSL:
 
 - ```powershell
   wsl --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz
```
Comment (SuperSandro2000) [inline README.md:25]: And the number below should now be 4

**Comment by andoriyu** [comment]
@SuperSandro2000 Makes sense. Made those changes.

File: README.md [hunk 3]
```diff
@@ -11,15 +11,21 @@ Modules for running NixOS on the Windows Subsystem for Linux
 
 ## Quick Start
 
-1. Download `nixos-wsl.tar.gz` from [the latest release](https://github.com/nix-community/NixOS-WSL/releases/latest).
+1. Enable WSL if you haven’t done so:
```
Comment (SuperSandro2000) [inline README.md:14]:
```suggestion
1. Enable WSL if you haven't done already:
```

**Pull request merged**